.claude/skills/beads-issue-tracking.md

```
@@ -0,0 +1,97 @@
+bd status                            # DB health summary
+```
+
+## Status Values
+
+`open` | `in_progress` | `blocked` | `done` | `closed`
+
+## Commit Convention
+
+Reference beads in commit messages:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)